components/library/qt5/Makefile

```diff
@@ -0,0 +1,117 @@
+
+QT_ARCHSUBDIR.32=
+QT_ARCHSUBDIR.64=$(MACH64)
+QT_PLATFORM.32= solaris-g++
+QT_PLATFORM.64= solaris-g++-64
+
+CONFIGURE_OPTIONS = -prefix $(QT_PREFIX)
+CONFIGURE_OPTIONS+= -verbose
+CONFIGURE_OPTIONS+= -opensource
+CONFIGURE_OPTIONS+= -confirm-license
+CONFIGURE_OPTIONS+= -nomake examples
+CONFIGURE_OPTIONS+= -nomake tests
+CONFIGURE_OPTIONS+= -c++std c++11
+CONFIGURE_OPTIONS+= -disable-reduce-exports
+CONFIGURE_OPTIONS+= -qt-xcb
+CONFIGURE_OPTIONS+= -bindir $(QT_PREFIX)/bin/$(QT_ARCHSUBDIR.$(BITS))
+CONFIGURE_OPTIONS+= -libdir $(QT_PREFIX)/lib/$(QT_ARCHSUBDIR.$(BITS))
+CONFIGURE_OPTIONS+= -plugindir $(QT_PREFIX)/plugins/$(QT_ARCHSUBDIR.$(BITS))
+CONFIGURE_OPTIONS+= -importdir $(QT_PREFIX)/imports/$(QT_ARCHSUBDIR.$(BITS))
+CONFIGURE_OPTIONS+= -qmldir $(QT_PREFIX)/qml/$(QT_ARCHSUBDIR.$(BITS))
+CONFIGURE_OPTIONS+= -platform $(QT_PLATFORM.$(BITS))
+CONFIGURE_OPTIONS+= -plugin-sql-mysql
+CONFIGURE_OPTIONS+= -plugin-sql-psql
+CONFIGURE_OPTIONS+= -system-sqlite
+CONFIGURE_OPTIONS+= $(CONFIGURE_OPTIONS.$(BITS))
+
+COMPONENT_INSTALL_ENV+=	INSTALL_ROOT=$(PROTO_DIR)
+
+# Set relative runpath to sql plugins
+QT_LIBDIR    =$(QT_PREFIX)/lib/$(QT_ARCHSUBDIR.$(BITS))
+QT_SQLDIR    =$(QT_PREFIX)/plugins/$(QT_ARCHSUBDIR.$(BITS))/sqldrivers
+QT_LIBRPATH  =$(shell realpath -m --relative-to=$(QT_SQLDIR) $(QT_LIBDIR))
+QT_MYSQLRPATH=$(shell realpath -m --relative-to=$(QT_SQLDIR) $(MYSQL_LIBDIR.$(BITS)))
+QT_PGSQLRPATH=$(shell realpath -m --relative-to=$(QT_SQLDIR) $(PG_LIBDIR.$(BITS)))
+
+COMPONENT_POST_INSTALL_ACTION= ( \
+/usr/bin/elfedit -e 'dyn:runpath "$$ORIGIN/$(QT_LIBRPATH):$$ORIGIN/$(QT_MYSQLRPATH)"' \
+	$(PROTO_DIR)$(QT_SQLDIR)/libqsqlmysql.so; \
+/usr/bin/elfedit -e 'dyn:runpath "$$ORIGIN/$(QT_LIBRPATH):$$ORIGIN/$(QT_PGSQLRPATH)"' \
+	$(PROTO_DIR)$(QT_SQLDIR)/libqsqlpsql.so; \
+)
+
+build:		$(BUILD_32_and_64)
+
+install:	$(INSTALL_32_and_64)
+
+REQUIRED_PACKAGES += codec/jasper
+REQUIRED_PACKAGES += database/mariadb-55/library
+REQUIRED_PACKAGES += database/sqlite-3
+REQUIRED_PACKAGES += image/library/libjpeg6
+REQUIRED_PACKAGES += image/library/libjpeg6-ijg
+REQUIRED_PACKAGES += image/library/libpng16
+REQUIRED_PACKAGES += image/library/libtiff
+REQUIRED_PACKAGES += library/audio/gstreamer
+REQUIRED_PACKAGES += library/audio/gstreamer/plugin/base
+REQUIRED_PACKAGES += library/audio/openal
+REQUIRED_PACKAGES += library/audio/pulseaudio
+REQUIRED_PACKAGES += library/c++/harfbuzz
+REQUIRED_PACKAGES += library/desktop/gtk3
+REQUIRED_PACKAGES += library/desktop/pango
+REQUIRED_PACKAGES += library/glib2
+REQUIRED_PACKAGES += library/print/cups-libs
+REQUIRED_PACKAGES += library/zlib
+REQUIRED_PACKAGES += system/library
+REQUIRED_PACKAGES += system/library/fontconfig
+REQUIRED_PACKAGES += system/library/freetype-2
+REQUIRED_PACKAGES += system/library/g++-4-runtime
+REQUIRED_PACKAGES += system/library/gcc-4-runtime
+REQUIRED_PACKAGES += system/library/libdbus
+REQUIRED_PACKAGES += system/library/math
+REQUIRED_PACKAGES += x11/keyboard/xkeyboard-config 
```

Comment:
pkg install: The following pattern(s) did not match any allowable packages.  Try
using a different matching pattern, or refreshing publisher information:
        /x11/keyboard/xkeyboard-config